src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -122,12 +122,14 @@ WHERE LOWER(u.email) ILIKE ANY (array[<emails>])
       """)
   List<User> findUsersByEmailList(@BindList("emails") List<String> emails);
 
-  @SqlUpdate("INSERT INTO users (email, display_name, create_date) values (:email, :displayName, :createDate)")
+  @SqlUpdate("INSERT INTO users (email, display_name, institution_id, create_date) values (:email, :displayName, :institutionId, :createDate)")
   @GetGeneratedKeys
   Integer insertUser(@Bind("email") String email,
       @Bind("displayName") String displayName,
+      @Bind("institutionId") Integer institutionId,
       @Bind("createDate") Date createDate);
```

Comment:
Since in most case this refactor sets `institutionId` to be `null`, would it make sense to create a `insertUserWithInstitution` instead make `insertUser` not take an institution as the default? (or removing the bind and explicitly setting `institutionId` to `null` in the query for clarity?)

Reply:
All the cases that use `null` are in test code, and I'd rather not add test-only code to the deployment.

I should be able to refactor test uses so there's only one or two cases where `null` is passed in.